achievements/console.md

```diff
@@ -1,9 +1,11 @@
 # Symfony Console
 
-In the Symfony Console component, we've fixed some bugs that were preventing
-some contributors to run the tests independently from each other and to
-contribute even more to the component.
+We fixed some bugs in the Symfony Console component including:
```

Comment:
It seems we lose the elements of the removed paragraph, since they are not present afterwards.